lib/openstack_query/queries/server_query.py

```diff
@@ -138,5 +114,6 @@ def _get_client_side_handlers(self) -> QueryClientSideHandlers:
         )
 
     def __init__(self):
-        self.marker_enum = ServerProperties.SERVER_ID
-        super().__init__(runner_cls=ServerRunner)
+        self.prop_enum_cls = ServerRunner
+        self.runner = ServerRunner(self.prop_enum_cls)
+        super().__init__()
```

Comment:
Also since this is the same for each class we should use the superclass to do this for us

```
class QueryWrapper(...):
   def __init__(prop_enum_cls: Class):
        # Note you're missing this in query wrapper at the moment, so that needs adding back
        super().__init__(...)
        self.prop_enum_cls = prop_enum_cls
       # I'm a bit confused why we pass the class name into the constructor for the class
        self.runner = prop_enum_cls(self.prop_enum_cls)
```

Reply:
I can move this into the superclass :+1: